example.config.yml

```diff
@@ -10,6 +10,15 @@ vagrant_hostname: drupalvm.dev
 vagrant_machine_name: drupalvm
 vagrant_ip: 192.168.88.88
 
+# If you need to access this guest from a device other than the host you will
+# want to enable public networking. Optionally provide a static IP value.
+# Warning! Vagrant boxes are insecure by default and by design, featuring public
+# passwords, insecure keypairs for SSH access, and potentially allow root access
+# over SSH.
+# https://docs.vagrantup.com/v2/networking/public_network.html
+vagrant_enable_public_network: false
```

Comment:
Let's simplify this a bit. Instead of a bool variable + the IP variable, let's do something like:

```
# Allow Drupal VM to be accessed via a network interface on your host by setting
# the IP address here. Vagrant boxes are insecure by default, so be careful
# exposing Drupal VM to the outside world. You've been warned!
vagrant_public_ip: ""
```

Then, see my notes in the logic in the Vagrantfile for the other side of this.